src/snowcli/cli/nativeapp/artifacts.py

```diff
@@ -234,3 +262,51 @@ def build_bundle(
             else:
                 # refuse to map multiple source files to one destination (undefined behaviour)
                 raise TooManyFilesError(dest_path)
+
+
+def find_manifest_file(deploy_root: Path) -> Path:
+    """
+    Find manifest.yml file, if available, in the deploy_root of the native apps project.
+    """
+    resolved_root = deploy_root.resolve()
+    for root, dirs, files in os.walk(resolved_root):
+        for file in files:
+            if file == "manifest.yml":
+                return Path(os.path.join(root, file))
+
+        for subdir in dirs:
+            # TODO: can the manifest.yml ever be in a subdir under deploy_root? It belongs at the root of the stage, so maybe not required.
```

Comment:
nope, not allowed: https://docs.snowflake.com/en/developer-guide/native-apps/creating-manifest#about-the-manifest-file